include/wlr/render/pixman.h

```diff
@@ -9,9 +9,24 @@
 #ifndef WLR_RENDER_PIXMAN_H
 #define WLR_RENDER_PIXMAN_H
 
+#include <pixman.h>
 #include <wlr/backend.h>
 #include <wlr/render/wlr_renderer.h>
 
 struct wlr_renderer *wlr_pixman_renderer_create(void);
+/**
+ * Returns the image of current buffer.
+ */
+pixman_image_t *wlr_pixman_renderer_get_current_image(
+	struct wlr_renderer *wlr_renderer);
+
+struct wlr_pixman_texture_attribs {
+	pixman_image_t *image;
+};
+
+bool wlr_renderer_is_pixman(struct wlr_renderer *wlr_renderer);
+bool wlr_texture_is_pixman(struct wlr_texture *texture);
+void wlr_pixman_texture_get_attribs(struct wlr_texture *wlr_texture,
```

Comment:
One last comment: can we make this `wlr_pixman_texture_get_image` and return a `pixman_image_t` directly instead of having a struct with only one field?

Reply:
Done